src/bundle/Resources/public/scss/_main-row.scss

```diff
@@ -0,0 +1,32 @@
+.ez-main-row {
+    display: grid;
+    grid-template-columns: 1fr calculateRem(80px);
+    grid-template-rows: auto;
+    grid-template-areas: "ez-content-container ez-context-menu";
+
+    .ez-content-container {
+        grid-area: ez-content-container;
+    }
+
+    .ez-context-menu {
+        grid-area: ez-context-menu;
+    }
+}
+
+.ez-has-two-sidebars {
```

Comment:
It should go into the declaration of `.ez-main-row` defined above this one:
```
.ez-main-row {
    ...
    .ez-has-two-sidebars & {
        ...
    }
```

The same applies to the `ez-dashboard` which should be renamed to `ez-has-no-sidebars`.
Keep in mind, to add such modifiers at the end of `.ez-main-row` declaration.

Reply:
I don't think that `ez-has-two-sidebars` is a good name for block, because it sounds like a modifier (considering BEM methodology). Maybe `ez-two-sidebar-view` (where I consider _two-sidebar_ as single word, adjective), or `ez-main-row--has-two-sidebars`?

Reply:
Thank you @sunpietro